include/uniTestingcpp/GameContext.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <uniTestingcpp/Board.h>
+#include <uniTestingcpp/Cell.h>
```

Comment:
#should use a different name for the container folder; this looks like actual game logic but the folder name hints it's related to unit tests